### DISCUSSION TITLE
What am I doing wrong?

### DISCUSSION THREAD

**Discussion by coatsy35** [Courseplay/Courseplay_FS22#2312] [General]
Two things. 

1. I setup a course for a combine or implement with 2 headland runs. I start at the nearest point, the combine goes fine around the outside and then shifts onto the second run without the fouter run being completed. Everytime I use headland mode. Not sure what I'm not doing correctly. 
2. Articulated T9 tractor on wheels, using straight corners, the tractor does not back up and pull the corners and runs out straight like a rigid tractor would o with say a trailed sprayer. IS this a known issue with articulated tractors? never seen it with a rigid tractor. 

I'm using the latest copy of courseplay.

**Comment by Tensuko** [comment]
1) You may use start on the first waypoint to be sure the start area is worked on as well. Else make a screenshot on where is the problem.

2) Got to try it out whats wrong, shouldnt be a problem normaly. What exact sprayer do you use ?

What Version do you use? "The latest" can be anything.

**Reply by coatsy35** [reply]
Hi version v7.2.0.1

I figured out point 1, I was being dumb, need to press start at the start. Is there a way to move the start?

as for point 2, the issue is with the New Holland T9.700 on dual wheels and the 9m Lizard mounted plough. It just waggle's about and doesn't do very neat corners even if corner is on straight. I never have this issue with fixed wheel tractors and items like a trailed sprayer, they work perfectly.

**Reply by Tensuko** [reply]
Check ingame help on how to use the course editor.
For the tractor I check what we can do.

**Reply by coatsy35** [reply]
Thank you. The worst field was weirdly Field 59 on elmcreek, 2 headlands and miss 1 row with the T9 and 9m plough. It was doing lots of erratic things.
